tests/sentry/api/test_event_search.py

```diff
@@ -99,6 +99,15 @@ def node_visitor(token):
         if token["type"] == "keyExplicitNumberTag":
             return SearchKey(name=f"tags[{token['key']['value']},number]")
 
+        if token["type"] == "keyExplicitFlag":
+            return SearchKey(name=f"flags[{token['key']['value']}]")
+
+        if token["type"] == "keyExplicitStringFlag":
+            return SearchKey(name=f"flags[{token['key']['value']},string]")
+
+        if token["type"] == "keyExplicitNumberFlag":
+            return SearchKey(name=f"flags[{token['key']['value']},number]")
+
```

Comment:
prob doesn't belong in this PR?

Reply:
This is supposedly an e2e test.  I'm not exactly sure how it works.  Its a bit complex.